src/ci/docker/x86_64-gnu-tools/checkregression.py

```diff
@@ -4,6 +4,9 @@
 import sys
 import json
 
+# Regressions for these tools do not cause failure.
```

Comment:
If what @kennytm said is accurate, this *only* means "do not cause failure during the beta cutoff week, but still *does* cause failure if a PR changing the submodule does not fix the tool". I have no idea how that's actually what happens here, but the comment should reflect reality.

Reply:
This py file has two purposes: 1. Detect if any tool *regressed*, and 2. Detect if any tool state *changed*. 

If the toolstate didn't change, we skip committing to the toolstate repo. 

If the toolstate did change, *and* we are in the beta week, *and* turns out the toolstate regressed too, we abort the job. 

You could determine how to best express this 😅

Reply:
That makes a lot of sense! Would be good to add it as a comment in some reasonable place. :)